pegasus/src/social_metadata.rb

```diff
@@ -20,124 +31,124 @@ def get_social_metadata_for_page(request)
-        image: {path: "/images/social-media/codeorg-2018-creativity.jpg", width: 846, height: 529},
+        image: images[:creativity],
         video: videos[:creativity_is]
       },
       "actual-hoc-dance" => {
         title: I18n.t(:og_title_here),
         description: hoc_s(:social_hoc2018_every_student_try_what_create),
-        image: {path: "/images/social-media/codeorg-2018-creativity.jpg", width: 846, height: 529},
+        image: images[:creativity],
         video: videos[:creativity_is]
       },
       "default" => {
         title: hoc_s(:social_hoc2018_code_org_what_create),
         description: hoc_s(:social_hoc2018_every_student),
-        image: {path: "/images/social-media/codeorg-2018-creativity.jpg", width: 846, height: 529},
+        image: images[:creativity],
         video: videos[:creativity_is]
       }
     },
     "csedweek.org" => {
       "soon-hoc" => {
         title: I18n.t(:csedweek_og_title),
         description: I18n.t(:csedweek_og_description_soon),
-        image: {path: "/images/cs-is-everything-thumbnail.png", width: 1200, height: 627},
+        image: images[:cs_is_everything_thumbnail],
         video: videos[:computer_science_is_changing_everything]
       },
       "actual-hoc" => {
         title: I18n.t(:og_title_here),
         description: I18n.t(:csedweek_og_description_here),
-        image: {path: "/images/cs-is-everything-thumbnail.png", width: 1200, height: 627},
+        image: images[:cs_is_everything_thumbnail],
         video: videos[:computer_science_is_changing_everything]
       },
       "default" => {
         title: I18n.t(:csedweek_og_title),
         description: I18n.t(:csedweek_og_description),
-        image: {path: "/images/cs-is-everything-thumbnail.png", width: 1200, height: 627},
+        image: images[:cs_is_everything_thumbnail],
         video: videos[:computer_science_is_changing_everything]
       }
     },
     "hourofcode.com" => {
       "soon-hoc" => {
         title: hoc_s(:social_hoc2018_hoc_coming_create),
         description: hoc_s(:social_hoc2018_creativity_in_are_you),
-        image: {path: "/images/social-media/hourofcode-2018-creativity.jpg", width: 846, height: 529},
+        image: images[:hoc_2018_creativity],
         video: videos[:creativity_is]
       },
       "actual-hoc" => {
         title: hoc_s(:social_hoc2018_hoc_here),
         description: hoc_s(:social_hoc2018_creativity_in_are_you),
-        image: {path: "/images/social-media/hourofcode-2018-creativity.jpg", width: 846, height: 529},
+        image: images[:hoc_2018_creativity],
         video: videos[:creativity_is]
       },
       "default" => {
         title: hoc_s(:social_hoc2018_join),
         description: hoc_s(:social_hoc2018_creativity_in_are_you),
-        image: {path: "/images/social-media/hourofcode-2018-creativity.jpg", width: 1440, height: 900},
+        image: images[:hoc_2018_creativity_large],
```

Comment:
Why do we have a small and large version?

Reply:
Ah, looks like I did it to match the dimensions of the prior image used here, but that's not useful because the source image is not that big.  I'll fix.